packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -366,6 +390,18 @@ class AndroidCameraCameraX extends CameraPlatform {
     previewInitiallyBound = true;
     _previewIsPaused = false;
 
+    // Retrieve info required for correcting the rotation of the camera preview
+    // if necessary.
+    isUsingSurfaceTextureForPreview =
+        await SystemServices.isUsingSurfaceTextureForPreview();
+    final Camera2CameraInfo camera2CameraInfo =
+        await proxy.getCamera2CameraInfo(cameraInfo!);
+    sensorOrientation = await proxy.getSensorOrientation(camera2CameraInfo);
+    naturalOrientation ??= await proxy.getUiOrientation();
+    onDeviceOrientationChanged().listen((DeviceOrientationChangedEvent event) {
```

Comment:
Will this stream need to be cancelled at some point/otherwise leak if not?

Reply:
Canceled the subscription!